_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+
+```
+opensearch-reporting-cli 
+```
+
+Upon success, you will see "Report downloaded" in the command line.
+-->
+
+## Get help
+
+To get a list of all available CLI arguments, run the following command.
+
+``` 
+$ opensearch-reporting-cli -h
+```
+
+
+## Reporting CLI options
+
+You can use any of the following arguments with the `opensearch-reporting-cli` tool.
+
+| Argument      | Description       | Acceptable values and usage | Environment variable
+:--------------------- | :--- | :--- |
+`-u`, `--url` | URL for the Dashboard visualization. | Obtain from OpenSearch Dashboards > Visualize > Share > Permalinks > Copy link. | URL
+`-a`, `--auth` | Authentication type for the report. | You can specify either Basic `basic`, Cognito `cognito`, SAML `saml` or No Auth `none`. If no value is specified, it defaults to No auth. Cognito and SAML require credentials with `-c` flag. | N/A
+`-c`, `--credentials` | OpenSearch login credentials. | Enter username and password separated by a colon. For example, username:password. Required for Cognito and SAML auth types. | USERNAME and PASSWORD
+`-t`, `--tenant` | Tenants in OpenSearch dashboards. | The default tenant is global.| N/A
+`-f`, `--format` | File format for the report. | Can be either `pdf`, `png`, or `csv`. The default is `pdf`.| N/A
+`-w`, `--width` | Window width in pixels for the report. | Default is `1680`.| N/A
+`-l`, `--height` | Minimum window height in pixels for the report. | Default is `600`. | N/A
+`-n`, `--filename` | File name of the report. | Default is `reporting`. | FILENAME
+`-e`, `--transport` | Transport for sending the email. | For Amazon SES specify `ses`. SES requires an AWS config on your system to store the credentials. For SMTP use `smtp` and also specify the login credentials with `--smtpusername` and `--smtppassword`. | TRANSPORT
+`-s`, `--from` | The email address of the sender. | For example, `user@amazon.com`. | FROM
+`-r`, `--to` | The email address of the recipient. | For example, `user@amazon.com`. | TO
+`--smtphost` | The hostname of the SMTP server. | For example, `SMTP_HOST`. | SMTP_HOST
+`--smtpport` | The port for connection. | For example, `SMTP_PORT`. | SMTP_PORT
+`--smtpsecure` | Use TLS when connecting to server. | For example, `SMTP_SECURE`. | SMTP_SECURE
+`--smtpusername` | The SMTP username.| For example, `SMTP_USERNAME`. | SMTP_USERNAME
+`--smtppassword` | The SMTP password.| For example, `SMTP_PASSWORD`. | SMTP_PASSWORD
+`--subject` | Email Subject text encased in quotes. | Can be any string. The default is "This is an email containing your dashboard report". | SUBJECT
+`-h`, `--help` | Display list of optional arguments from the command line. | N/A
+
+
+
+## Use environment variables with Reporting CLI
+
+Instead of explicitly providing values in the command line, you can save them as environment variables. The Reporting CLI reads environment variables from .env file inside the project.
+
+```
+.
+Reporting
```

Comment:
Please remove file this structure. My doc isn't updated. `.env` file can be in current directory. Also `.env` file isn't necessary to set environment variables. e.g. on linux we can use
```
export NAME=VALUE
```